starter-code/routes/celebrities.js

```diff
@@ -0,0 +1,74 @@
+const express = require('express');
+const router  = express.Router();
+const Celebrity = require("../models/Celebrity.js");
+
+router.get("/celebrities" , (req, res, next) => {
+  Celebrity.find()
+    .then(data => {
+      res.render("celebrities/index", { data });
+    })
+    .catch(error => {
+      console.log(error);
+    });
+});
+router.get('/new', (req,res) => res.render('celebrities/new'));
+router.post("/celebrities", (req, res, next) => {
+  const {name,occupation,catchPrhase} = req.body;
+  const newCelebrity = new Celebrity({name,occupation,catchPrhase})
```

Comment:
You have a spelling mistake, that is why the `catchPhrase` is not been save in the database... 😉